usm: Added docker-swarm documentation (#16825)
DataDog/documentation: content/en/universal_service_monitoring/_index.md

     - /:/host/root:ro
```

{{% /tab %}}
{{% tab "Docker Swarm" %}}

> As `Docker Swarm` does not yet support of changing `security_opt`, then we can run only if the Operating System
> does not have a running `apparmor` instance.

Reviewer: ```suggestion
As `Docker Swarm` does not yet support the changing of `security_opt`, the operating system
must not have a running `apparmor` instance.
```

> If that is the case, then use the same `docker-compose.yml` file from `Docker-Compose` [section][1] besides the field `security_opt`.

Reviewer: ```suggestion
If the operating system does not have a running `apparmor` instance, then use the same `docker-compose.yml` file from `Docker-Compose` [section][1] beside the field `security_opt`.
```
